Magento_Catalog/templates/product/image_with_borders.phtml

```diff
@@ -2,7 +2,7 @@
 
 <div class="lazyload-wrapper">
     <img
-        class="image lazyload product-image-photo"
+        class="image catalog-grid-item__image lazyload"
```

Comment:
Please dont remove `product-image-photo`, it's used in `swatch-renderer` script

Reply:
It would be great to add the comment for what is this class

Reply:
Ok. Didn't know that. I will leave it with `.product-image-photo`